### PR TITLE
update escrow docs

### PR DESCRIPTION
**Pull request by scottrepreneur**
- update out of date deps (docusaurus v3)
- update docs structure

### CODE REVIEW

File: docs/raids/escrow/interacting-with-escrow.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+_who can do this? **resolver**_
+
+[resolve form image]
+
+- [Check out the docs for resolving on SmartInvoice](https://docs.smartinvoice.xyz/tutorials/arbitrator/release-disputed-escrow-funds)
+
+### Withdraw
+
+_who can do this? **client, provider**. when can you do this? **after safety valve date**_
```
Comment (plor) [inline docs/raids/escrow/interacting-with-escrow.md]: Isn't this just the client? When can the provider withdraw?

File: docs/raids/escrow/zap.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The zap transaction completes creating 3 - 4 different contracts, composing them together, in a single transaction. No more interface hopping to get started raiding. Check out the full diagram of the zap workflow.
+
+![flow image](/escrow/escrow-flow.png)
+
+Dungeon Master supports the 4 major flows of the zap:
+
+1. Create a safe + DAO split + raid party split + escrow (green)
+2. Create a safe + DAO split + escrow (blue)
+3. Create a safe + raid party split + escrow (orange)
+4. Create a safe + escrow
```
Comment (plor) [inline docs/raids/escrow/zap.md:22]: Does this include using an existing safe?

File: static/escrow/escrow-flow.png
Comment (plor) [inline static/escrow/escrow-flow.png:1]: Lots going on here, but looks great!

**Review by plor — APPROVED** [review]
Looks good. A couple comments, but nothing critical I think.

File: docs/raids/escrow/interacting-with-escrow.md (hunk 1 above)
Comment (scottrepreneur) [reply docs/raids/escrow/interacting-with-escrow.md]: A `provider` theoretically can withdraw after `resolve`? this might be pasta also

**Pull request merged**